api.py

```diff
@@ -1,114 +1,12 @@
-from flask import Flask, jsonify, abort, url_for
+from flask import Flask, abort
 from flask_restful import Api, Resource, reqparse, fields, marshal
+from resources.reservations import ReservationList, Reservation
```

Comment:
I'm getting an error on this line:
`ModuleNotFoundError: No module named 'resources'`

Reply:
When you run api.py from which directory are you running it? I can get it to work from hotel_api/...but I think I'm missing some best practices here in the way I have it currently configured.